src/login/components/LoginFormLedger/index.js

```diff
@@ -23,6 +23,9 @@ export default compose(
   withActions(ledgerActions, mapLedgerActionsToProps),
   withData(ledgerActions, mapLedgerDataToProps),
   withError(ledgerActions, mapLedgerErrorToProps),
+  withProgress(ledgerActions, {
+    progress: recentlyCompletedStrategy
```

Comment:
Here's the core issue.  The key should be `strategy`, not `progress`.  I've got some changes I'll push up and open a PR against your branch.

Reply:
Brilliant, thank you Matt! 
Sorry for wasting your time before.
You even wrote strategy in your change request after the first review, but I missed it...

Reply:
No problem!